src/main/java/Duke.java

```diff
@@ -1,10 +1,49 @@
+import java.util.Arrays;
+import java.util.Scanner;
+
 public class Duke {
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        String logo = " ╔╦═╦══╦═╦╗\n" +
+                      " ║╚╗║╔╗║═╣║\n" +
+                      " ║╔╣║╚╝║═╣║\n" +
+                      " ║╚═╩══╩═╝║\n" +
+                      " ╚════════╝\n";
+        System.out.println("Hiiii, my name is\n" + logo );
+        System.out.println("I am here to help :3\n");
+        String line;
+        String[] tasks = new String[100];
+        int nextTask = 0;
+        Scanner in = new Scanner(System.in);
+        System.out.print("Yes darling, what can i do for you?\n");
+        line = in.nextLine();
+        while (!line.equals("bye")){
+            if (line.equals("list")){
+                printTasks(tasks, nextTask);
+            }
+            if (!line.equals("list")) {
+                tasks[nextTask] = line;
+                nextTask++;
+                System.out.println("I just added " + line + " to your list!!!");
+            }
```

Comment:
The 'list' condition is checked twice unnecessarily. Consider replacing it with an if-else block
```suggestion
            if (line.equals("list")) {
                printTasks(tasks, nextTask);
            } else {
                tasks[nextTask] = line;
                nextTask++;
                System.out.println("I just added " + line + " to your list!!!");
            }
```